src/model/GeoW_FunctionsExt_23.ili

```diff
@@ -0,0 +1,67 @@
+
+    !!@ fn.description = "Gruppiert die Eingabemenge gemäss dem übergebenen Attribut-Kriterium / -Pfad. Für 'Attr' soll der Pfad zum klassierenden Attribut in INTERLIS 2 Syntax angegeben werden.";
+    !!@ fn.param = "Objects: Ausgangsobjekte. Attr: Pfad zum Attribut welches die Klassierung bestimmen soll";
+    !!@ fn.return = "Zugehörige Gruppe gemäss Attr des aktuellen Kontextobjekt";
+    !!@ fn.since = "2022-12-02";
+    FUNCTION GetInGroups (Objects: OBJECTS OF ANYCLASS; Attr: TEXT): OBJECTS OF ANYCLASS;
+
+    !!@ fn.description = "Prüft, ob ein Objekt innerhalb einer gegebenen Testgeometrie liegt, welche aus einem externen Datensatz stammt. Der externe Datensatz muss sich in einer INTERLIS 2.3 Transferdatei befinden, die entweder im selben Ordner wie die GeoW_FunctionsExt JAR liegt oder sich in einem JAR befinden, das über den Java Classpath erreichbar ist.";
+    !!@ fn.param = "DatasetName: Qualifizierter Name des Attributes aus dem Transfermodells, in welchem die Objekte zur Prüfung bereitliegen. Objects: Objekt(e), über die TID identifiziert, welche zur Prüfung beigezogen werden. TestObject: Objekt, welches zu prüfen ist. TestObjectgeometry: Geometriefeld, bezogen auf das unter Testobject übergebene Objekt";
+    !!@ fn.return = "Boolean";
+    !!@ fn.since = "2022-12-05";
+    FUNCTION IsInsideExternalDataset (DatasetName: TEXT; Objects: TEXT; TestObject: OBJECT OF ANYCLASS; TestObjectgeometry: TEXT): BOOLEAN;
+
+    !!@ fn.description = "Prüft, ob ein Objekt innerhalb einer gegebenen Testgeometrie liegt, welche aus einem externen Datensatz innerhalb der Ressourcen stammt. Der externe Datensatz muss sich in einer INTERLIS 2.3 Transferdatei befinden, die über den angegebenen Pfad als Ressource erreichbar ist.";
+    !!@ fn.param = "TransferFile: Qualifizierter Name der Transferdatei-Ressource. DatasetName: Qualifizierter Name des Attributes aus dem Transfermodell, in welchem die Objekte zur Prüfung bereitliegen. Objects: Objekt(e), über die TID identifiziert, welche zur Prüfung beigezogen werden. TestObject: Objekt, welches zu prüfen ist. TestObjectgeometry: Geometriefeld, bezogen auf das unter Testobject übergebene Objekt";
+    !!@ fn.return = "Boolean";
+    !!@ fn.since = "2023-12-20";
+    FUNCTION IsInsideExternalDatasetResource (TransferFile: TEXT; DatasetName: TEXT; Objects: TEXT; TestObject: OBJECT OF ANYCLASS; TestObjectgeometry: TEXT): BOOLEAN;
+
+    !!@ fn.description = "Prüft, ob ein Objekt innerhalb der Geometrie eines anderen Objektes liegt.";
+    !!@ fn.param = "ReferenceGeometry: Referenzgeometrie, innerhalb welcher das TestObject liegen muss. TestObject: Objekt, welches zu prüfen ist. TestObjectgeometry: Geometriefeld, bezogen auf das unter Testobject übergebene Objekt";
+    !!@ fn.return = "Boolean";
+    !!@ fn.since = "2023-12-21";
+    FUNCTION IsInside (ReferenceGeometry: GeometryCHLV95_V1.MultiSurface; TestObject: OBJECT OF ANYCLASS; TestObjectgeometry: TEXT): BOOLEAN;
```

Comment:
Q: Was ist der Grund, weshalb aus dem MULTIAREA ein MultiSurface und nicht ein GeometryCHLV95_V1.Multi _Area_ wurde?

Reply:
_MultiArea_ sollte es ja eigentlich gar nicht geben, der widerspricht sich wie in sich. Er ist ja auch im RefHB nur so halbherzig abgehandelt und wohl ein Kandidat zum Streichen.

Reply:
Q: Können die Funtkionen bereits mit den `GeometryCHLV95_V1` Geometrien umgehen?

Reply:
ja, aufgrund der Testergebnisse klappt das mit 2.3 Transfers und GeometryCHLV95_V1.MultiSurfaceals Returnwert